### ISSUE TITLE
fabric

### ISSUE THREAD

**Issue by meganrwong** [ANZSoilData/def-au-domain#14]
So earth, sandy rough ped and smooth peds are types of fabric, and they should be over in the collections of classifiers? 

re 'fabric' it's definition is like 'structure' it is clearly re soil-fabric. Unlike structure I'm not sure we need a high level fabric, ie fabric>soil-fabric though

**Comment by dr-shorthair** [comment]
Yes, they have codes on p182. 

Rock also has Fabric and Structure and descriptions of rock might appear in the same dataset. 
So Fabric and Structure should be managed at the same level. 

The definitions of Structure and Fabric are specific to soils. 
Not sure what the conclusion of all this is.

**Comment by meganrwong** [comment]
yes, the def of structure on p 171 is specific to soils - so all obsv prop from pgs 171-181 relate to soi stucture. However, there are some obs prop of structure that fall outside this, like soil-structure  p195. That's how I came at we need sprop:structure>sprop:soil-structure.
For fabric, no obsv prop fall outside of the obsv prop 'fabric' (which we could call 'soil-fabric'). 

Here are the obsv prop I think are narrower to a general 'structure'

![image](https://user-images.githubusercontent.com/49380601/184288739-4c8aee47-338c-416b-b052-4caf87c3ac9e.png)

then, these are under soil-structure

![image](https://user-images.githubusercontent.com/49380601/184288807-94c60bde-8b3a-47bc-a0e8-9c4438f28f7f.png)

**Comment by meganrwong** [comment]
#10 - structure follows same logic as colour
"actually I think we need obsv prop 'colour' and 'soil-colour'
because we also have mottle-colour to put in under colour.
I'd put the def of colour from yb (ie saying it is measured by munsell) as a note under soil-colour"
